dio/lib/src/adapters/browser_adapter.dart

```diff
@@ -76,14 +77,22 @@ class BrowserHttpClientAdapter implements HttpClientAdapter {
               StackTrace.current,
             );
             xhr.abort();
+          } else {
+            print(
+                'Warn: connectTimeout is triggered after fetch has completed.');
           }
```

Comment:
We should avoid `print` in the production code.

Reply:
Because this project does not depend on any logging library, so print is used.
How should I modify it? Add `if(kDebugModel)` or use `log()`?

Reply:
Can just ignore it?

Reply:
Or leaving this with a comment.

```suggestion
          } else {
            // connectTimeout is triggered after the fetch has been completed.
          }
```